packages/webapp/src/containers/revenueTypeSlice.js

```diff
@@ -117,5 +118,33 @@ export const allRevenueTypesSelector = createSelector(
   },
 );
 
-export const revenueTypeSelector = (revenue_type_id) => (state) =>
+export const revenueTypeByIdSelector = (revenue_type_id) => (state) =>
   revenueTypeSelectors.selectById(state, revenue_type_id);
+
+const sortRevenueTypes = (revenueTypes) => {
+  const defaultTypes = [];
+  const customTypes = [];
+
+  revenueTypes?.forEach((type) => {
+    const arrayToUpdate = type.farm_id ? customTypes : defaultTypes;
+    arrayToUpdate.push(type);
+  });
+
+  return [
+    ...defaultTypes.sort((typeA, typeB) =>
+      i18n
+        .t(`revenue:${typeA.revenue_translation_key}`)
+        .localeCompare(i18n.t(`expense:${typeB.revenue_translation_key}`)),
```

Comment:
Should this be

```suggestion
        .localeCompare(i18n.t(`revenue:${typeB.revenue_translation_key}`)),
```

?

Reply:
Yes thanks!